src/contracts/repositories/createContractsRepository.ts

```diff
@@ -0,0 +1,87 @@
+import { CarStatus, ContractsStatus } from '@prisma/client';
+
+import prisma from '../../lib/prisma.js';
+
+export const createContractsRepository = {
+  // 차량 정보 조회
+  findCarByIdForContract: async (carId: number) => {
+    const car = await prisma.cars.findUnique({
+      where: { id: carId },
+      select: {
+        id: true,
+        price: true, // 차량 가격
+        status: true, // 차량 상태
+        carModel: { select: { model: true } }, // 차량 이름
+      },
+    });
+    if (!car) throw new Error(`차량을 찾을 수 없습니다`);
```

Comment:
```suggestion
    if (!car) throw createError(404, '차량을 찾을 수 없습니다');
```

throw new Error로 보내면 에러를 전부 500으로 받지 않을까요? 전역 에러 핸들러가 있으니 사용하는 게 좋을 거 같아요